pkg/services/plugindashboards/plugindashboards_test.go

```diff
@@ -0,0 +1,233 @@
+package plugindashboards
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/grafana/grafana/pkg/bus"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/plugins"
+	"github.com/grafana/grafana/pkg/services/dashboardimport"
+	"github.com/stretchr/testify/require"
+)
+
+func TestService(t *testing.T) {
+	t.Run("Creating a new service should not update app dashboards when there's no plugins ", func(t *testing.T) {
```

Comment:
For this to be a valid test, should you not be mocking those calls and verifying that `pluginDashboardManager.GetPluginDashboards` is never called (for example)?

Reply:
Yeah good point. These tests is WIP. Have struggled a lot to get a structure that looks somewhat understandable in regards to the setup.

Reply:
Yeah it's hard when you gotta mock so much. I think in general it looks good though!

Reply:
Updated. Think it's much better and clearer now